internal/hud/server/server.go

```diff
@@ -175,6 +184,30 @@ func (s *HeadsUpServer) HandleAnalytics(w http.ResponseWriter, req *http.Request
 	}
 }
 
+func (s *HeadsUpServer) DispatchAction(w http.ResponseWriter, req *http.Request) {
```

Comment:
Interesting idea taking all of the action dispatchers in to one endpoint!

Are we worried about how non-homogeneous the inputs to this endpoint will be? What happens when `actionPayload` has a bunch of possible fields which are used for different actions that it can dispatch?

Reply:
nope! keep in mind that we can guarantee that the client and server are always in-sync. So it doesn't really matter. We can break backwards-compatibility